apps/blockchain/lib/blockchain/bloom.ex

```diff
@@ -1,4 +1,17 @@
 defmodule Blockchain.Bloom do
+  @moduledoc """
```

Comment:
@ayrat555 I see you wrote a post on the bloom filter - adding any of those comments or examples to this file may be useful, or at least a link to the article. http://www.badykov.com/ethereum/2017/10/29/ethereum-bloom-filter/

Reply:
@andogro good idea